<?xml version="1.0"?>
<launch>

  <arg name="generator_config_path" default="$(find pickplace_bridge)/config/serialGenConfig.yaml"/>
  <arg name="actuator_config_path"  default="$(find pickplace_bridge)/config/serialPPConfig.yaml"/>
  <arg name="move_group_config_path"  default="$(find pickplace_bridge)/config/serialPPMoveGroupConfig.yaml"/>

  <node pkg="pickplace_bridge" type="pickplace_bridge" name="pickplace_bridge" output="screen">
     <param name="generator_config_path" type="str" value="$(arg generator_config_path)"/>
     <param name="actuator_config_path" type="str" value="$(arg actuator_config_path)"/>
     <param name="move_group_config_path" type="str" value="$(arg move_group_config_path)"/>
  </node>

  <arg name="move_group_name"  default="arm" />
  <node pkg="gripper_bridge" type="serial_gripper_finger.py" name="serial_gripper"  output="screen"  >
    <param name="move_group_name" type="str" value="arm" />
  </node>	

</launch>
